cron/job.js

```diff
@@ -14,58 +14,59 @@ client.on("error", function (err) {
     console.error("Error " + err);
 });
 
-const superBlockCycle = energiUtil.getSuperBlockCycle()
 
 function populateProposalList() {
-    energiUtil.getGovernanceObjectList((err, list) => {
-        if (err) {
-            console.error(err);
-        } 
-        else if (list )
-        // var govArray = new Array();
-        var multi = client.multi()
-        var obj = list.result
-        for (var key in obj) {
-            isProposal = false
-            gobj = obj[key];
-            g = new Array();
-            data = gobj['DataString'];
-            json = JSON.parse(data);
-            for (var i in json) {
-                (function (i) {
-                    item = json[i];
-                    if (item[0] == 'proposal') {
-                        isProposal = true;
-                        let num = (item[1].end_epoch - item[1].start_epoch) / superBlockCycle
-                        g.push(item[1].name + ' ' + '(<a href="' + item[1].url + '">Details</a>)')
-                        g.push(parseInt(item[1].payment_amount) * num)
-                        g.push(num)
-                        let d = new Date(item[1].start_epoch) * 1000
-                        g.push(moment(d).utc().toString())
-                        d = new Date(item[1].end_epoch) * 1000
-                        g.push(moment(d).utc().toString())
-                    } else
-                        return
-                })(i);
-            }
+    energiUtil.getSuperBlockCycle(function(blockCycle) {
+        superBlockCycle = blockCycle;
+        energiUtil.getGovernanceObjectList((err, list) => {
+            if (err) {
+                console.error(err);
+            } else if (list)
+                // var govArray = new Array();
+                var multi = client.multi()
+            var obj = list.result
+            for (var key in obj) {
+                isProposal = false
+                gobj = obj[key];
+                g = new Array();
+                data = gobj['DataString'];
+                json = JSON.parse(data);
+                for (var i in json) {
+                    (function(i) {
+                        item = json[i];
+                        if (item[0] == 'proposal') {
+                            isProposal = true;
+                            let num = (item[1].end_epoch - item[1].start_epoch) / superBlockCycle
+                            g.push(item[1].name + ' ' + '(<a href="' + item[1].url + '">Details</a>)')
+                            g.push(parseInt(item[1].payment_amount) * num)
```

Comment:
@narek-h please remove the `* num` here to fix the payment amount, as well. The rest looks good. Thank you!

Reply:
@yghazi , thanks a lot for the review and the comment. Just want to share my understanding and confirm that should be changed. 
As I understand, we need the total budget there, so if we have 1000NRG and two payments,  we need to show '1000*2=2000', no ?